cppcache/src/TcrEndpoint.cpp

```diff
@@ -37,6 +37,8 @@ namespace geode {
 namespace client {
 
 const char* TcrEndpoint::NC_Notification = "NC Notification";
+volatile bool TcrEndpoint::TEST_DISCONNECTIONS = false;
```

Comment:
If we are going to use a global variable, or even if this becomes and instance variable on `CacheImpl` we should use `std::atomic_flag` here.

Good writeup on why here: https://www.drdobbs.com/parallel/volatile-vs-volatile/212701484

Reply:
Thanks for the article!